tRecorderApi/api/models/project.py

```diff
@@ -1,458 +1,453 @@
 import json
 import os
 from django.db import models
-from django.forms.models import model_to_dict
-from .chunk import Chunk
 
 
 class Project(models.Model):
     version = models.ForeignKey("Version")
     mode = models.ForeignKey("Mode", on_delete=models.CASCADE)
     anthology = models.ForeignKey("Anthology")
     language = models.ForeignKey("Language")
-    source_language = models.ForeignKey(
-        "Language",
-        related_name="language_source",
-        blank=True
-    )
+    # source_language = models.ForeignKey(

```

Comment:
why is this commented out?

Reply:
I was going to ask why do they have the same foreign key? ( my plan was to comment it so I would not forget to ask... ) I guess this way works too